### PR TITLE
Chatie devops

### PR DESCRIPTION
**Pull request by huan**
See #150

1. This PR did not change the code a lot, it only refactors the DevOps pipeline
1. It can pass  `npm run test:unit` without any problem.
1. It can not pass `npm run lint` because the linting rules had been changed a bit, but it is very easy to be fixed with the helpful tips from VSCode.

The next steps from my suggestion are:

1. merge this PR first
1. fix all the linting problems
1. commit and push
1. make CI turns green

If you have any questions please feel free to reply to the issue to discuss.

### CODE REVIEW

**Comment by huan** [comment]
P.S. to fix this PR is a very good job for new developers.

File: src/config.ts [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 /// <reference path="./typings.d.ts" />
 
-import Brolog from 'brolog'
-import readPkgUp from 'read-pkg-up'
+import { log } from 'brolog'
```
Comment (windmemory) [inline src/config.ts]: What's the reason for removing the separate logger of wechaty-puppet-padpro?

Comment (huan) [reply src/config.ts]: For ... saving memory ...

Indeed, it's an unrelated minor change, if we need a separate logger for padpro, please feel free to just revert this change.

Comment (windmemory) [reply src/config.ts]: Okay, let's keep this "memory saving" change.

**Comment by windmemory** [comment]
Looks good, let's merge this and start working on the lint fix.

**Comment by windmemory** [comment]
And thanks for the PR, you are the best @huan

**Comment by huan** [comment]
You are welcome!

**Pull request merged**